roles/openshift_node/tasks/dnsmasq_install.yml

```diff
@@ -18,6 +18,25 @@
   until: result is succeeded
   when: not openshift_is_atomic | bool
 
+- name: Create override dnsmasq systemd unit directory
+  file:
+    path: /etc/systemd/system/dnsmasq.service.d
+    state: directory
+  when: not openshift_is_atomic | bool
+
+- name: Copy systemd dnsmasq unit override file
+  copy:
+    src: config/override.conf
+    dest: /etc/systemd/system/dnsmasq.service.d/override.conf
+  when: not openshift_is_atomic | bool
+
+- name: Restart dnsmasq service and reload configuration
+  systemd:
+    state: restarted
```

Comment:
You shouldn't need to restart dnsmasq in this role—`roles/openshift_node/tasks/dnsamsq.yml` is run after `dnsmasq_install.yml`, and `dnsmasq.yml` restarts dnsmasq.

Come to think of it, all of these changes might belong here, or they might belong in `dnsmasq.yml`; the division between `dnsmasq_install.yml` and `dnsmasq.yml` is not clear to me.

Reply:
good point